README.md

```diff
@@ -1,7 +1,7 @@
 # scala-steward-public-repos
 _Configuration for the Guardian's in-house instance of Scala Steward, suitable for PUBLIC Guardian repos_
 
-If you have a **public** Guardian repo you'd like to scan with Scala Steward, **request to [install the GitHub app](https://github.com/apps/gu-scala-steward-public-repos) on your repository** - this repo (`scala-steward-public-repos`) is public,
+If you have a **public** Guardian repo you'd like to scan with Scala Steward, **request to [install the GitHub app](https://github.com/apps/gu-scala-steward-public-repos) on your repository and add it to the list in [REPOSITORIES.md](REPOSITORIES.md)** - this repo (`scala-steward-public-repos`) is public,
```

Comment:
Maybe link out to https://github.com/scala-steward-org/scala-steward-action#updating-a-custom-branch so folk can read the docs if they wish to update a custom branch?